lib/nerves_hub/repo.ex

```diff
@@ -64,3 +64,28 @@ defmodule NervesHub.Repo do
 
   def destroy(struct_or_changeset), do: delete(struct_or_changeset)
 end
+
+defmodule NervesHub.ObanRepo do
+  use Ecto.Repo,
+    otp_app: :nerves_hub,
+    adapter: Ecto.Adapters.Postgres
+
+  alias NervesHub.Config
+
+  @doc """
+  Dynamically loads the repository url from the
+  DATABASE_URL environment variable.
+  """
+  def init(_, config) do
+    vapor_config = Vapor.load!(Config)
+    database_config = vapor_config.database
+
+    config =
+      Keyword.merge(config,
+        pool_size: database_config.pool_size,
+        url: database_config.url
+      )
+
+    {:ok, config}
+  end
```

Comment:
```suggestion
  defdelegate init(context, config), to: NervesHub.Repo
```
If the goal is to keep it the same, would you want to just delegate here so it uses the same init codepath?